classes/DirtOrbit.sc

```diff
@@ -173,22 +175,24 @@ DirtOrbit {
 			~overgain = 0.0;
 			~cut = 0.0;
 			~unit = \r;
-			~n = \none; // sample number or note
-			~octave = 5;
-			~midinote = #{ ~note ? ~n + (~octave * 12) };
+			~n = nil;
```

Comment:
we should find a way around setting this to nil. Why can't it be `\none`?

Reply:
Basically, you can just directly replace the default if you need, no need for StrudelUtils:

```
~dirt.set(\n, 42); // whatever you want
```